### PR TITLE
SPM VBM pipeline

### PR DESCRIPTION
**Pull request by glatard**
This adds functions to perform an SPM VBM analysis from a notebook through the following sequence:
* `spm_compute_missing_segmentations`
* `spm_compute_dartel_normalization`
* `spm_compute_intra_cranial_volumes`
* `spm_compute_vbm_stats`
A Pandas object containing a LivingPark cohort is the main parameter of these functions. The functions parametrize SPM batch templates and run them in a container using Boutiques.

### CODE REVIEW

**Comment by glatard** [comment]
Hello @mathdugre, do you have any particular comment on this before we merge this PR and release a new version?

**Comment by mathdugre** [comment]
Hi @glatard, I did not have the time to have a look at it yet. I'll have time to look at it over the weekend.

**Comment by glatard** [comment]
Thanks, no rush, it can wait until you have 30' on a work day. No need to review the details of the pipeline itself, I am more looking for your feedback on the packaging (though it passes the pre-commit hooks so I guess it's ok).

File: livingpark_utils/livingpark_utils.py [hunk 1]
```diff
@@ -785,5 +802,320 @@ def qc_spm_segmentations(self, cohort) -> None:
...
+
+        Parameters
+        ----------
+         cohort: pd.DataFrame
+            A LivingPark cohort. Must have columns PATNO and EVENT_ID
+         tissue_class: int
+            "gm" for grey matter, "wm" for white matter
+         group1/2_patnos: list of PPMI patient numbers in group1/2. Important: a patno
+            must appear in exactly one group and must appear excatly once in this group.
+        icys: dict
```
Comment (mathdugre) [inline livingpark_utils/livingpark_utils.py:1065]: Variable name is inconsistent with the function signature.

**Review by mathdugre — COMMENTED** [review]
A few nitpicks here and there. Should be good to merge afterwards.

**Pull request merged**